CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 10.3.0 (YYYY-MM-DD)
+
+### Enhancements
+* None.
+
+### Fixes
+* None.
```

Comment:
We should probably add a comment about https://jira.mongodb.org/browse/HELP-21241. Maybe something like `* [RealmApp] Sometimes getting `Index out of range error` when integrating a remote Sync changeset into the local Realm`

It is a bit vague, but not sure I can come up with something more specific that doesn't involve writing 10s of lines of text.